kratos/utilities/auxilary_global_master_slave_relation.h

```diff
@@ -221,6 +221,55 @@ class AuxilaryGlobalMasterSlaveRelation : public IndexedObject
 
 }; // End of ConstraintEquation class
 
+/**
+ * @class LocalIndices
+ * @ingroup KratosCore
+ * @brief This class stores the stores three different vectors of local internal, slave, master indices
+ *          which are used in constraint builder and solver.
+ *
+ * @author Aditya Ghantasala
+ */
+class LocalIndices
+{
+    public:
+    KRATOS_CLASS_POINTER_DEFINITION(LocalIndices);
+    typedef std::size_t IndexType;
+    typedef std::vector<IndexType> VectorIndexType;
+
+    /**
+     * Structure "LocalIndices" to be used as an encapsulator for set of local elemental/conditional indices
+    */
+    LocalIndices()
+    {
+        internal_index_vector.resize(0);
+        master_index_vector.resize(0);
+        slave_index_vector.resize(0);
+    }
+
+    ~LocalIndices()
+    {
+        internal_index_vector.resize(0);
```

Comment:
is this needed? Won't those vectors be destroyed anyways?
They are not shared from what I see